### ISSUE TITLE
feat: Add app icon in the console

### ISSUE THREAD

**Issue by feinstein** [shorebirdtech/shorebird#2459]
I would like to add app icons in my "Shorebird Apps" list.

I manage a bunch of apps in an agency and it makes it easier to visually find the app I want if I can add app icons to the "Shorebird Apps" list. Codemagic has a similar feature.

**Comment by eseidel** [comment]
Makes sense.  We do seem to have a lot of "white label app" customers with lots of apps.
